chain/chainhandle.go

```diff
@@ -953,12 +954,12 @@ func executeTx(execCtx context.Context, ccc consensus.ChainConsensusCluster, cdb
 		}
 	case types.TxType_FEEDELEGATION:
 		balance := receiver.Balance()
-		var fee *big.Int
-		fee, err = tx.GetMaxFee(balance, bs.GasPrice, bi.ForkVersion)
+		var maxFee *big.Int
+		maxFee, err = fee.TxMaxFee(bi.ForkVersion, len(tx.GetBody().GetPayload()), tx.GetBody().GetGasLimit(), balance, bs.GasPrice)
```

Comment:
I don't understand why to do this if the original line is simpler

Reply:
I want to separate transaction and fee calculation logic.
so, code will be longer because it need to extract the value from tx .